Jenkinsfile

```diff
@@ -1,96 +1,21 @@
 #!/usr/bin/env groovy
 
-REPOSITORY = 'frontend'
-DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
-
 node('mongodb-2.4') {
```

Comment:
I think this might be added here by mistake previously, frontend doesn't use mongo.